src/commit.c

```diff
@@ -419,6 +418,10 @@ int git_commit__parse_commit(git_commit *commit, git_odb_object *odb_obj, git_co
 		}
 	}
 
+	/* Perform necessary grafts */
+    git_repository_odb__weakptr(&odb, repo);
+    if (odb)
+        git_odb__graft_object(commit, odb);
```

Comment:
Huh? Either it is GitHub which confused the commit ordering or this commit is too early in that it introduces using `git_odb__graft_object` previous to it being implemented.